meson.build

```diff
@@ -35,13 +35,23 @@ if is_freebsd
 	add_project_arguments('-D_C11_SOURCE', language: 'c')
 endif
 
+# Execute the wlroots subproject, if any
+wlroots_version = ['>=0.16.0', '<0.17.0']
+subproject(
```

Comment:
When did meson allowed to this?

Reply:
0.54.0, since that's when `meson.override_dependency` was introduced.

Reply:
The syntax feels a bit weird. So this will automatically use the dependency in `subprojects` when calling `dependency` if the wlroots subproject is there?

Reply:
Yeah: executing the subproject will override the "wlroots" dep.

Reply:
Good to know!